### PR TITLE
Fix import react router

### PR DESCRIPTION
**Pull request by arthurepc**
#### What is the purpose of this pull request?

Add file to export react-router-dom library.

#### What problem is this solving?

Encapsulate react-router-dom for all my-account apps, so there will be no more version problems

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/Router.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import * as ReactRouterDom from 'react-router-dom'
+
+/*
+  IMPORTANT: READ BEFORE MAKING ANY EDITIONS IN THIS FILE
+
+  This file's goal is to make the use of react-router-dom
+  unified through all the apps within the My Account ecosystem.
+  Delete this if a better solution to the problem of different
```
Comment (lbebber) [inline react/Router.js]:
```suggestion
  Delete this if a better solution to the problem or different
```

File: react/Router.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  versions comes up.
```
Comment (lbebber) [inline react/Router.js]:
```suggestion
  versions come up.
```

File: react/Router.js (hunk 1 above)
Comment (arthurepc) [reply react/Router.js]: The ideia here is to refer to the problem's characteristics. Why do you think using 'or' would be a good fit here?

Comment (lbebber) [reply react/Router.js]: ah, I thought it was a typo. nvm

**Pull request merged**